crates/services/upgradable-executor/src/executor.rs

```diff
@@ -113,7 +113,7 @@ impl<S, R> Executor<S, R> {
     /// This constant is used along with the `version_check` test.
     /// To avoid automatic bumping during release, the constant uses `-` instead of `.`.
     #[cfg(test)]
-    pub const CRATE_VERSION: &'static str = "0-25-2";
+    pub const CRATE_VERSION: &'static str = "0-25-3";
```

Comment:
Since we change only GraphQL API, we don't need to increase the version

Reply:
Reverted